src/popup/router/components/Header.vue

```diff
@@ -109,109 +114,138 @@ export default {
 @import '../../../styles/typography';
 
 .header {
+  width: 360px;
   position: fixed;
   top: 0;
-  left: 0;
-  right: 0;
   z-index: 8;
-  height: calc(50px + env(safe-area-inset-top));
+  height: calc(48px + env(safe-area-inset-top));
   background-color: $color-black;
   display: flex;
-  justify-content: center;
+  padding: 8px 16px 8px 8px;
+  padding-top: calc(8px + env(safe-area-inset-top));
   align-items: center;
 
-  .content {
-    flex: 1;
-    max-width: $container-width;
-    height: 50px;
-    padding: 0 10px;
-    padding-top: env(safe-area-inset-top);
+  .left {
     display: flex;
-    align-items: center;
+    flex-basis: 88px;
+  }
+
+  .right {
+    display: flex;
+    flex-basis: 82px;
+    justify-content: flex-end;
+  }
+
+  .title {
+    min-width: 166px;
+    flex-grow: 2;
 
-    .left,
-    .right {
-      flex-basis: 80px;
+    .text {
+      padding: 0 4px;
       display: flex;
-      align-items: center;
-    }
+      justify-content: center;
+
+      @extend %face-sans-16-medium;
 
-    .right {
-      justify-content: flex-end;
+      line-height: 24px;
+      color: $color-white;
     }
+  }
 
-    .title {
-      flex: 1 0;
+  .left .home-button {
+    padding: 4px 0;
+    height: 32px;
 
-      .text {
-        max-width: 180px;
-        padding: 0 4px;
-        display: flex;
-        justify-content: center;
+    &:not(:disabled) {
+      svg {
+        cursor: pointer;
+      }
 
-        @extend %face-sans-16-medium;
+      &:hover svg {
+        color: $color-blue-hover;
+      }
 
-        color: $color-white;
+      &:active svg {
+        color: $color-blue-hover;
+        opacity: 0.9;
       }
     }
 
-    .logo {
-      width: 36px;
-      height: 26px;
+    svg {
+      width: 34px;
+      height: 24px;
+      color: $color-blue;
     }
+  }
 
-    .back {
-      margin-left: 10px;
-    }
+  .home-button + .back {
+    margin-left: 22px;
+  }
```

Comment:
```suggestion
  .home-button + .back {
    margin-left: 22px;
  }
```
Adding this because of the case where is no `home-button` like in  `ImportAccount` page